Trippy/Views/SelectableMap.m

```diff
@@ -13,6 +13,11 @@
 @interface SelectableMap () <GMSMapViewDelegate>
 @property (strong, nonatomic) GMSMapView *mapView;
 @property (strong, nonatomic) NSMutableArray *markersArray;
+@property (strong, nonatomic) GMSPath *path;
+@property (strong, nonatomic) GMSMutablePath *animatedPath;
+@property (strong, nonatomic) GMSPolyline *animatedPolyline;
+@property int i;
```

Comment:
What is i? It's acceptable in a for loop, but as a class property, it really needs to be more descriptive. Looking below, something like `curAnimPathIndex` would work. I would also explicitly specify the property attributes - if nothing else, it doesn't look like everyone else here.